aws/cloudformation/bootstrap_chef_stack.sh.erb

```diff
@@ -11,9 +11,9 @@ unattended-upgrade
 apt-get -y install python3-pip curl
 test "$(pip3 show awscli)" || pip3 install awscli
 
-STACK=<%=stack_name%>
+STACK=${AWS::StackName}
 CHEF_VERSION=12.7.2
-REGION=<%=region%>
+REGION=${AWS::Region}
```

Comment:
This is the syntax for `Fn::Sub` references. The rest of the changes to shell scripts replace `${VAR}` syntax with `$VAR` to avoid colliding with the `Fn::Sub` syntax.